src/version3/models/newUserDetails.ts

```diff
@@ -4,4 +4,10 @@ export interface NewUserDetails {
   self?: string;
   /** The email address for the user. */
   emailAddress: string;
+  /**
+   * Products the new user has access to. Valid products are: jira-core, jira-servicedesk, jira-product-discovery,
+   * jira-software. To create a user without product access, set this field to be an empty array.
+   * Defaults to ['jira-core', 'jira-servicedesk', 'jira-product-discovery', 'jira-software'].
+   */
+  products?: string[];
```

Comment:
Let's refine the typings to enhance IDE suggestions:

```suggestion
  products?: ('jira-core' | 'jira-servicedesk' | 'jira-product-discovery' | 'jira-software' | string)[];
```